hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -462,7 +461,6 @@ protected void waitForLeaderToBeReady()
     // Wait for Leader Election timeout
     int timeout = OZONE_OM_RATIS_SERVER_FAILURE_TIMEOUT_DURATION_DEFAULT
         .toIntExact(TimeUnit.MILLISECONDS);
```

Comment:
`timeout` is now unused.  Please remove it, as well as the import of `OZONE_OM_RATIS_SERVER_FAILURE_TIMEOUT_DURATION_DEFAULT`.

Reply:
Fixed it re-commited.